substrafl/strategies/strategy.py

```diff
@@ -93,10 +89,57 @@ def predict(
         test nodes.
 
         Args:
-            algo (Algo): algo with the code to execute on the organization
             test_data_nodes (typing.List[TestDataNode]): list of nodes on which to evaluate
             train_data_nodes (typing.List[TrainDataNode]): list of nodes on which the model has
                 been trained
             round_idx (int): index of the round
         """
         raise NotImplementedError
+
+    def build_graph(
+        self,
+        train_data_nodes: List[TrainDataNode],
+        aggregation_node: Optional[List[AggregationNode]],
+        evaluation_strategy: Optional[EvaluationStrategy],
+        num_rounds: int,
+        clean_models: Optional[bool],
+    ):
+        """_summary_
+
```

Comment:
I think I would add a comment in the doc string that the graph built is "stored" by side-effect in each train_data_nodes / aggregation_nodes, so that it's easier to understand what is happening in the function without reading all the code base.

Reply:
Wooops you're right, completly forgot the docstring !

Reply:
👍